addons/hr_timesheet/security/ir.model.access.csv

```diff
@@ -3,5 +3,5 @@ access_account_analytic_line_user,analytic.account.analytic.line.timesheet.user,
 access_account_analytic_user,analytic.account.analytic.timesheet.user,analytic.model_account_analytic_account,hr_timesheet.group_hr_timesheet_user,1,1,0,0
 access_uom_uom_hr_timesheet,uom.uom.timesheet.user,uom.model_uom_uom,hr_timesheet.group_hr_timesheet_user,1,0,0,0
 access_project_project,project.project.timesheet.user,model_project_project,hr_timesheet.group_hr_timesheet_user,1,0,0,0
-access_project_task,project.task.timesheet.user,model_project_task,hr_timesheet.group_hr_timesheet_user,1,1,0,0
+access_project_task,project.task.timesheet.user,model_project_task,hr_timesheet.group_hr_timesheet_user,1,0,0,0

```

Comment:
After discussing with Alexandra (AJU), we will do it in master and in master, we will completely remove this line since internal user already had read access.

Don't forget to do a migration script for that changes.

Thanks in advance! 🙂